snappi_ixnetwork/device/ngpf.py

```diff
@@ -120,6 +120,8 @@ def _configure_device_group(self, ixn_topos):
         port_name = None
         for device in self.api.snappi_config.devices:
             for ethernet in device.get("ethernets"):
+                if ethernet.get("connection") and ethernet.get("port_name"):
+                    raise Exception("Both attributes: connection and port_name is passed for device {}".format(device.get("name")))
```

Comment:
put a more appropriate message like `port_name is deprecated and usage of vxlan_name in connection and port_name` is not allowed.